### PR TITLE
docs: document eraser flags

### LINKED ISSUE #290: document eraser pod flags
**Describe the solution you'd like**
Looks like we are missing these in the docs:

```
	successDelDelaySeconds = flag.Int64("job-cleanup-on-success-delay", 0, "Seconds to delay job deletion after successful runs. 0 means no delay")
	errDelDelaySeconds     = flag.Int64("job-cleanup-on-error-delay", 86400, "Seconds to delay job deletion after errored runs. 0 means no delay")
	successRatio           = flag.Float64("job-success-ratio", 1.0, "Ratio of successful/total runs to consider a job successful. 1.0 means all runs must succeed.")
```

### PR DESCRIPTION
**Pull request by ashnamehrotra**
**What this PR does / why we need it**:
Adds documentation for eraser pod flags with delays changed from seconds to duration.

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #290 

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/docs/eraser-flags.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Eraser Pod Flags
```
Comment (sozercan) [inline docs/docs/eraser-flags.md]: `Customization` maybe?

File: docs/docs/eraser-flags.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Customization
+---
+
```
Comment (sozercan) [inline docs/docs/eraser-flags.md]: let's add this is for eraser specifically. can these be configured through manager?

Comment (sozercan) [reply docs/docs/eraser-flags.md]: if not, i think we need a way similar to how scanner args are passed through

Comment (ashnamehrotra) [reply docs/docs/eraser-flags.md]: they can be passed in like the scanner args, but it applies to both eraser and collector right now. Should we remove the delay for collector pods?

**Review by pmengelbert — APPROVED** [review]

File: docs/docs/eraser-flags.md (hunk 2 above)
Comment (ashnamehrotra) [reply docs/docs/eraser-flags.md]: addressed eraser/collector pods in #313

File: docs/docs/eraser-flags.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (sozercan) [inline docs/docs/eraser-flags.md]: can we call the file customization.md too?

**Pull request merged**